src/views/Home.jsx

```diff
@@ -2,8 +2,12 @@ import { Navigate } from 'react-router-dom';
 import './Home.css';
 import { generateToken } from '@the-collab-lab/shopping-list-utils';
 import * as React from 'react';
+import { validateToken } from '../api/firebase';
+import Toastify from 'toastify-js';
+import 'toastify-js/src/toastify.css';
 
 export function Home({ setListToken }) {
+	const [userTokenInput, setUserTokenInput] = React.useState('');
```

Comment:
Suggesting again - import `{ useState } from 'react'` as opposed to `React.useState`. Not a blocker, but a bit cleaner IMO.

Reply:
Whoops, forgot about this! Done. Definitely cleaner!